etcd/src/test/java/org/jclouds/etcd/features/KeysApiLiveTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.etcd.features;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+import java.util.UUID;
+
+import org.jclouds.etcd.BaseEtcdApiLiveTest;
+import org.jclouds.etcd.domain.keys.Key;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "KeysApiLiveTest", singleThreaded = true)
+public class KeysApiLiveTest extends BaseEtcdApiLiveTest {
+
+   private String key;
+   private String value;
+
+   @BeforeClass
+   protected void init() {
+      key = UUID.randomUUID().toString().replaceAll("-", "");
+      value = UUID.randomUUID().toString().replaceAll("-", "");
+   }
+
+   @Test
+   public void testCreateKeyWithTTL() {
+      String localKey = UUID.randomUUID().toString().replaceAll("-", "");
+      String localValue = UUID.randomUUID().toString().replaceAll("-", "");
+      Key createdKey = api().createKey(localKey, localValue, 1);
+      assertNotNull(createdKey);
+      assertNotNull(createdKey.node().expiration());
+
+      try {
+         Thread.sleep(3000);
+      } catch (InterruptedException e) {
+         e.printStackTrace();
+      }
+
+      createdKey = api().getKey(localKey);
+      assertNull(createdKey);
+   }
+
+   @Test(dependsOnMethods = "testCreateKeyWithTTL")
```

Comment:
This test uses different keys than the TTL one. There is no need to depend on it.